### ISSUE TITLE
experiment replication

### ISSUE THREAD

**Issue by hunzhizi** [James-QiuHaoran/LLM-serving-with-proxy-models#11]
Hello, while replicating the experiment, the model's accuracy is not very satisfactory. Could you please tell me the number of epochs used in the first stage (full-parameter training) and the second stage (fixing the weights of BERT-base)?

**Comment by James-QiuHaoran** [comment]
Number of epochs is set to [6](https://github.com/James-QiuHaoran/LLM-serving-with-proxy-models/blob/main/output-token-len-prediction/latency_prediction.py#L487) during training. In the first 3 epochs, the BERT-base weight is updated but frozen after that (only updating the classification layer).

The `data_size` parameter also matters (see [issue](https://github.com/James-QiuHaoran/LLM-serving-with-proxy-models/issues/10) here). The example training command uses a small data size of 10 only for demonstration purposes. For full evaluation, please consider set the data size to be the training dataset size.
